### ISSUE TITLE
Removing teams and archiving repos

### ISSUE THREAD

**Issue by Trott** [nodejs/admin#686]
I'd like to remove the following teams and archive their associated repos:

* @nodejs/social This will mean transferring management of our Twitter account to the OpenJS Foundation. @richardlau pointed out that https://github.com/nodejs/tweet is referenced in https://github.com/nodejs/node/blob/master/doc/contributing/security-release-process.md as part of the security release process. The process for this cab cgabge but we'll need to remember to update the documented process. There is likely additional automation around the tweets repo or other things that probably only a handful of people (@bnb, @joesepi, probably a few others) are aware of.
* @nodejs/nodejs-collection 
* @nodejs/user-feedback 

/ping @nodejs/tsc

**Comment by bnb** [comment]
I'm curious why? As far as I know, neither of the two active members of the Social Team have been engaged in talking about this and it's rather unprecedented to recommend removing an active team without talking about it with those team members.

I'm disappointed in our processes here because I feel I know the reason why and feel like this is a borderline personal attack that is dishing out a consequence based off a dispute that didn't go through normal processes, and frankly feel extremely disappointed that I've received zero outreach or support in this. It is extremely othering and not in line with the project I know and love.

**Comment by WaleedAshraf** [comment]
@Trott We should also archive https://github.com/nodejs/social-media-delegates repo. The initiative isn't active anymore.

**Comment by dshaw** [comment]
+1 for archiving the @nodejs/user-feedback team. It hasn't been active since I stepped down 3 years ago.

**Comment by Trott** [comment]
Since removing `user-feedback` seems non-controversial, I'll do that now, but proceed a bit more slowly on the others.

**Comment by bnb** [comment]
I'll leave my final notes about the Social Team: 

I'm disappointed that rather than anyone who was engaging in apparent back-room discussions that were critical of the Social Team trying to engage with the Social Team - either by talking to or working with the team or by actually stepping up to do the work - the decision was made to propose it be removed. I genuinely hope we can try to do better with other teams in the future, as this is a fundamental departure from how we've historically managed _any_ other team as far as I'm aware.

I stand by my statement earlier that this feels like indirect (intentional or not) validation of one side an unresolved project-level interpersonal issue and a way to avoid actually meaningfully addressing that. We _will_ still need to solve that communication issue - removing the Social Team does not remove the tension, and it will just surface in other unhealthy and dysfunctional ways until it's resolved.

With regards to what should be done with the accounts:

I personally believe that my participation in anything surrounding the project's Social Media will only cause grief and pain - both for myself and for the project.  I am deeply saddened by this, as it's both something I care deeply about and frankly something I feel I did a good job at given the tools and information I had available to me.

I fundamentally do not object to the proposal given that belief - I encourage it.

I would _strongly recommend_ that the TSC effectively _lease_ the social media accounts to the OpenJS Foundation rather than handing them over wholesale. More specifically:

- The TSC should retain the ability to send out their own messages, if they want to and at their discretion.
- The TSC should retain the ability to augment or remove content that is on the project's accounts, if they want to and at their discretion.
- The TSC should retain the ability to allow or disallow certain kinds of content on the project's accounts, if they want to and at their discretion.
- The TSC should retain the ability to reclaim full control of the project's accounts, if they want to and at their discretion.

I assert the above because, fundamentally, the project's goals and the Foundations goals aren't _necessarily_ always the same - this is intrinsic of an open source project existing under a Foundation, and isn't unique to Node.js. Social Media is a massively impactful tool for both parties in this dynamic, especially with the scale at which the project's accounts exist. We should _not_ lose the ability to have ownership of our voice - this is something that's happened with other similarly impactful projects in other spaces, and their maintainership has _deeply_ regretted it. I want to ensure we never have those regrets.

It is entirely reasonable for the project to set boundaries with the Foundation and have them be respected without question, given that it is _our_ voice being used.

As we proceed, I hope the above recommendation is taken into consideration and adopted. Feel free to reach out to me directly on Slack, but I'm going to disengage from this thread unless my continued engagement is required.
